extmod/uasyncio/core.py

```diff
@@ -215,20 +234,41 @@ def run(coro):
 
 
 class Loop:
-    def create_task(self, coro):
+    @staticmethod
```

Comment:
Note that making the functions all staticmethod's will increase RAM usage because each use of the staticmethod decorator allocates a block in the heap.  Since RAM is precious I think it's worth trying to optimise it here and not use `staticmethod` if at all possible.

Reply:
Hmm I see, I changed everything back to not use staticmethods.
The downside is, that every exception now first needs to create a Loop() instance before the exception handler can be called.

Reply:
In #5841 I've removed the need to create a `Loop()` instance.